docs/cli.md

```diff
@@ -94,10 +98,79 @@ With `--all` flag, you create config file including all default setting of offic
       executable_methods:
         - method_name: "node_governance_alarm"`
 ```
-
 Also, You can set the home directory of VATZ (default "~/.vatz") with `--home` flag.
 And you can create the new config file with your disired name with `--output` flag
 
+## Start
+You can start Vatz when you finish set up config default.yaml or your own config yaml file.
```

Comment:
@xellos00 

It seems like a good idea to change it to VATZ for word uniformity.

Reply:
> @xellos00
> 
> It seems like a good idea to change it to VATZ for word uniformity.


Yeah, That's right. I agree!! I will make everything to `VATZ`